<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
<class>GD_Qly</class>
<widget class="QMainWindow" name="GD_Qly">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QUẢN LÝ NHÂN SỰ</string>
  </property>
  <property name="windowIcon">
   <iconset resource="icons.qrc">
    <normaloff>:/icons/app-icon.svg</normaloff>:/icons/app-icon.svg</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">/* --- Global Styles --- */
QWidget {
    font-family: &quot;Segoe UI&quot;, Arial, sans-serif;
font-size: 10pt;
    color: #333;
}

QMainWindow {
    background-color: #f0f2f5;
}

/* --- Navigation Sidebar --- */
#navFrame {
    background-color: #ffffff;
    border-right: 1px solid #e0e0e0;
}

#navFrame QPushButton {
    text-align: left;
    padding: 12px 20px;
    border: none;
    border-radius: 8px;
    background-color: transparent;
    font-weight: 500;
}

#navFrame QPushButton:hover {
    background-color: #f0f2f5;
}

#navFrame QPushButton:checked {
    background-color: #e6f2ff;
    color: #005a9e;
    font-weight: bold;
}

#headerLabel {
    font-size: 14pt;
    font-weight: bold;
    color: #005a9e;
    padding: 10px;
}


/* --- Content Area --- */
#page_employees, #page_attendance, #page_projects {
    background-color: #f0f2f5;
}

QGroupBox {
    font-weight: bold;
    background-color: #ffffff;
    border: 1px solid #dcdcdc;
    border-radius: 8px;
    margin-top: 10px;
}
QGroupBox::title {
    subcontrol-origin: margin;
    subcontrol-position: top left;
    padding: 0 10px;
    left: 10px;
}

/* --- General Widgets --- */
QLineEdit, QComboBox {
    border: 1px solid #dcdcdc;
    border-radius: 8px;
    padding: 8px 12px;
background-color: white;
}
QLineEdit:focus, QComboBox:focus {
    border: 1px solid #0078d7;
}

QPushButton {
    padding: 8px 18px;
border: 1px solid #dcdcdc;
    border-radius: 8px;
    background-color: white;
    min-width: 80px;
}
QPushButton:hover {
    background-color: #f5f5f5;
    border-color: #c0c0c0;
}
QPushButton:pressed {
    background-color: #e0e0e0;
}


/* --- Specialized Buttons --- */
QPushButton#btnAdd {
    background-color: #0078d7;
color: white;
    border-color: #0078d7;
    font-weight: bold;
}
QPushButton#btnAdd:hover {
    background-color: #005a9e;
    border-color: #004578;
}

QPushButton#btnDelete {
    background-color: #d9534f;
    color: white;
    border-color: #d9534f;
}
QPushButton#btnDelete:hover {
    background-color: #c9302c;
    border-color: #ac2925;
}


/* --- Table --- */
QTableWidget {
    border: 1px solid #dcdcdc;
    border-radius: 8px;
    gridline-color: #e0e0e0;
    background-color: white;
}
QHeaderView::section {
    background-color: #f8f9fa;
    padding: 10px;
    border: none;
    border-bottom: 1px solid #dcdcdc;
    font-weight: bold;
}
QTableWidget::item {
    padding: 10px;
}
QTableWidget::item:selected {
    background-color: #cfe2ff;
    color: #000;
}

/* --- ScrollBar --- */
QScrollBar:vertical {
    border: none;
    background: #f0f2f5;
    width: 10px;
    margin: 0px 0px 0px 0px;
}
QScrollBar::handle:vertical {
    background: #c0c0c0;
    min-height: 20px;
    border-radius: 5px;
}
QScrollBar::add-line:vertical, QScrollBar::sub-line:vertical {
    height: 0px;
}
</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_main">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QFrame" name="navFrame">
      <property name="minimumSize">
 
      <size>
        <width>220</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>220</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
  
    <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="leftMargin">
        <number>10</number>
       </property>
       <property name="topMargin">
        <number>10</number>
       </property>
       <property name="rightMargin">
        <number>10</number>
       </property>
     
  <property name="bottomMargin">
        <number>10</number>
       </property>
       <item>
        <widget class="QLabel" name="headerLabel">
         <property name="text">
          <string>QUẢN LÝ</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
      
  </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnNavEmployees">
         <property name="text">
          <string>Quản lý Nhân viên</string>
         </property>
         <property name="icon">
          <iconset resource="icons.qrc">
           <normaloff>:/icons/users.svg</normaloff>:/icons/users.svg</iconset>
         </property>
 
        <property name="checkable">
          <bool>true</bool>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
         <property name="autoExclusive">
          <bool>true</bool>
         </property>
         <property name="toolTip">
    
      <string>Quản lý hồ sơ và thông tin nhân viên</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnNavAttendance">
         <property name="text">
          <string>Kiểm soát Chấm công</string>
         </property>
         <property name="icon">
   
       <iconset resource="icons.qrc">
           <normaloff>:/icons/calendar.svg</normaloff>:/icons/calendar.svg</iconset>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <property name="autoExclusive">
          <bool>true</bool>
         </property>
         <property name="toolTip">
    
      <string>Theo dõi và quản lý dữ liệu chấm công</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnNavProjects">
         <property name="text">
          <string>Quản lý Dự án</string>
         </property>
         <property name="icon">
   
       <iconset resource="icons.qrc">
           <normaloff>:/icons/briefcase.svg</normaloff>:/icons/briefcase.svg</iconset>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <property name="autoExclusive">	
          <bool>true</bool>
         </property>
         <property name="toolTip">
    
      <string>Phân công và theo dõi tiến độ các dự án</string>
         </property>
        </widget>
       </item>
<item>
 <widget class="QPushButton" name="btnNavDepartments">
  <property name="text">
   <string>Quản lý Phòng ban</string>
  </property>
  <property name="icon">
   <iconset resource="icons.qrc">
    <normaloff>:/icons/office.svg</normaloff>:/icons/office.svg</iconset>
  </property>
  <property name="checkable">
   <bool>true</bool>
  </property>
  <property name="autoExclusive">
   <bool>true</bool>
  </property>
  <property name="toolTip">
   <string>Quản lý thông tin và danh sách phòng ban</string>
  </property>
 </widget>
</item>
 
      <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
   
      </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="btnNavSettings">
         <property name="text">
          <string>Cài đặt</string>
         </property>
         <property name="icon">
          <iconset resource="icons.qrc">
           
<normaloff>:/icons/settings.svg</normaloff>:/icons/settings.svg</iconset>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <property name="autoExclusive">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   
 <item>
     <widget class="QStackedWidget" name="mainStackedWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="page_employees">
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <property name="leftMargin">
         <number>12</number>
        </property>
        <property name="topMargin">
         <number>12</number>
        </property>
   
     <property name="rightMargin">
         <number>12</number>
        </property>
        <property name="bottomMargin">
         <number>12</number>
        </property>
        <item>
         <widget class="QFrame" name="topBarFrame">
          <layout class="QHBoxLayout" name="topBar">
           <item>
         
   <widget class="QLineEdit" name="lineEditSearch">
             <property name="minimumSize">
              <size>
               <width>350</width>
               <height>0</height>
              </size>
             </property>
          
   <property name="placeholderText">
              <string>Tìm kiếm theo tên, mã nhân viên...</string>
             </property>
             <property name="clearButtonEnabled">
              <bool>true</bool>
             </property>
            </widget>
          
 </item>
           <item>
            <spacer name="horizontalSpacer">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
     
          <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QPushButton" 
name="btnExport">
             <property name="text">
              <string>Xuất Excel</string>
             </property>
             <property name="icon">
              <iconset resource="icons.qrc">
               <normaloff>:/icons/export.svg</normaloff>:/icons/export.svg</iconset>
             </property>
 
           </widget>
           </item>
           <item>
            <widget class="QPushButton" name="btnRefresh">
             <property name="text">
              <string>Làm mới</string>
             </property>
           
  <property name="icon">
              <iconset resource="icons.qrc">
               <normaloff>:/icons/refresh.svg</normaloff>:/icons/refresh.svg</iconset>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
    
    <item>
         <widget class="QFrame" name="actionFrame">
          <layout class="QHBoxLayout" name="actionLayout">
           <item>
            <widget class="QPushButton" name="btnAdd">
             <property name="text">
              <string>Thêm Mới</string>
             </property>
      
       <property name="icon">
              <iconset resource="icons.qrc">
               <normaloff>:/icons/add.svg</normaloff>:/icons/add.svg</iconset>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="btnEdit">
 
            <property name="text">
              <string>Sửa</string>
             </property>
             <property name="icon">
              <iconset resource="icons.qrc">
               <normaloff>:/icons/edit.svg</normaloff>:/icons/edit.svg</iconset>
             </property>
   
         </widget>
           </item>
           <item>
            <widget class="QPushButton" name="btnDelete">
             <property name="text">
              <string>Xóa</string>
             </property>
             <property 
name="icon">
              <iconset resource="icons.qrc">
               <normaloff>:/icons/delete.svg</normaloff>:/icons/delete.svg</iconset>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="btnDetail">
         
    <property name="text">
              <string>Chi Tiết</string>
             </property>
             <property name="icon">
              <iconset resource="icons.qrc">
               <normaloff>:/icons/detail.svg</normaloff>:/icons/detail.svg</iconset>
             </property>
          
  </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_2">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
       
       <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
   
      </widget>
        </item>
        <item>
         <widget class="QTableWidget" name="tableEmployees">
          <property name="editTriggers">
           <set>QAbstractItemView::NoEditTriggers</set>
          </property>
          <property name="alternatingRowColors">
           <bool>true</bool>
          </property>
   
       <property name="selectionMode">
           <enum>QAbstractItemView::SingleSelection</enum>
          </property>
          <property name="selectionBehavior">
           <enum>QAbstractItemView::SelectRows</enum>
          </property>
          <property name="sortingEnabled">
           <bool>true</bool>
          </property>
       
   <attribute name="horizontalHeaderStretchLastSection">
           <bool>true</bool>
          </attribute>
          <column>
           <property name="text">
            <string>Mã NV</string>
           </property>
          </column>
          <column>
         
  <property name="text">
            <string>Họ và Tên</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Giới Tính</string>
           </property>
          </column>
      
    <column>
           <property name="text">
            <string>Phòng Ban</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Chức Vụ</string>
           </property>
    
      </column>
          <column>
           <property name="text">
            <string>Mức Lương</string>
           </property>
          </column>
         </widget>
        </item>
       </layout>
      </widget>
      
      <widget class="QWidget" name="page_attendance">
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <property name="spacing">
         <number>10</number>
        </property>
        <property name="leftMargin">
         <number>12</number>
        </property>
        <property name="topMargin">
         <number>12</number>
        </property>
        <property name="rightMargin">
         <number>12</number>
        </property>
        <property name="bottomMargin">
         <number>12</number>
        </property>
        <item>
         <widget class="QFrame" name="frame_filter_chamcong">
          <property name="frameShape">
           <enum>QFrame::NoFrame</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <property name="leftMargin">
            <number>0</number>
           </property>
           <property name="topMargin">
            <number>0</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <item>
            <widget class="QLabel" name="label_chonNV">
             <property name="text">
              <string>Chọn nhân viên:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="comboChamCongNhanVien">
             <property name="minimumSize">
              <size>
               <width>250</width>
               <height>0</height>
              </size>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_chonNgay">
             <property name="text">
              <string>Chọn ngày:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCalendarWidget" name="calendarChamCong">
             <property name="maximumSize">
              <size>
               <width>400</width>
               <height>300</height>
              </size>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="btnLoadChamCong">
             <property name="text">
              <string>Tải dữ liệu</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_3">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QTableWidget" name="tableChamCongTongHop">
          <property name="editTriggers">
           <set>QAbstractItemView::NoEditTriggers</set>
          </property>
          <property name="selectionMode">
           <enum>QAbstractItemView::SingleSelection</enum>
          </property>
          <property name="selectionBehavior">
           <enum>QAbstractItemView::SelectRows</enum>
          </property>
          <property name="sortingEnabled">
           <bool>true</bool>
          </property>
          <attribute name="horizontalHeaderStretchLastSection">
           <bool>true</bool>
          </attribute>
          <column>
           <property name="text">
            <string>Mã NV</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Họ và Tên</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Giờ vào</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Giờ ra</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Trạng thái</string>
           </property>
          </column>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="page_projects">
       <layout 
class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Nội dung cho trang Quản lý Dự án</string>
          </property>
          <property name="font">
           <font>
            <pointsize>16</pointsize>
       
    </font>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
  
  <rect>
     <x>0</x>
     <y>0</y>
     <width>1280</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="icons.qrc"/>
 </resources>
 <connections/>
</ui>